_rules/table-header-cell-has-assigned-cells-d0f69e.md

```diff
@@ -327,27 +334,12 @@ This `th` element is not a descendant in the [flat tree][] of an element with a
 </div>
 ```
 
-#### Inapplicable Example 7
```

Comment:
I think we can keep the example but update the description to point out that the `table` element does not have a role of `table`, so the rule does not apply.